app/src/main/res/values/strings.xml

```diff
@@ -36,6 +36,8 @@
     <string name="attributes_picker_rack_type_label">Which type of rack is in the parking spot?</string>
     <string name="attributes_picker_has_surveillance_question">Is the parking spot under camera surveillance ?</string>
     <string name="attributes_picker_description_label">Description (Optional)</string>
+    <string name="parking_added_with_reward_toast">Parking spot added!\n+100 coins for contributing to the community!</string>
```

Comment:
respect the nomenclature of string : "name=screen_name_variable_name"

Reply:
100 Should be a global variables declared in my opinion in the wallet.kt and used here